<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="soccf-tr5-001" name="sequential-ui-tests(failed)(failed)(failed)(failed)" tests="1" failures="1" timestamp="15 Sep 2014 10:51:09 GMT" time="29.349" errors="0">
  <testcase name="allTests" time="14.51" classname="teammates.test.cases.ui.browsertests.InstructorCommentsPageUiTest">
    <failure type="java.lang.AssertionError" message="URL: http://localhost:8888/page/instructorCommentsPage?user=comments.idOfInstructor1OfCourse1&amp;amp;courseid=comments.idOfTypicalCourse1">
      <![CDATA[java.lang.AssertionError: URL: http://localhost:8888/page/instructorCommentsPage?user=comments.idOfInstructor1OfCourse1&courseid=comments.idOfTypicalCourse1
	at teammates.test.cases.ui.browsertests.InstructorCommentsPageUiTest.testScripts(InstructorCommentsPageUiTest.java:96)
	at teammates.test.cases.ui.browsertests.InstructorCommentsPageUiTest.allTests(InstructorCommentsPageUiTest.java:33)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- allTests -->
</testsuite> <!-- sequential-ui-tests(failed)(failed)(failed)(failed) -->
